includes.container/usr/bin/host-shell

```diff
@@ -12,7 +12,8 @@ fi
 if [ "$(basename "${0}")" != "host-shell" ]; then
     host_command="$(basename "${0}")"
 else
-    host_command="${1:-${SHELL:-/bin/sh}}"
+    host_shell=$(grep "^$USER" /run/host/etc/passwd | awk -F: '{print $NF}')
+    host_command="${1:-$host_shell}"
```

Comment:
This should still have a fallback in the (unlikely) case that extracting the shell from passwd doesn't work